packages/dbos-cloud/applications/update-app.ts

```diff
@@ -0,0 +1,42 @@
+import axios, { AxiosError } from "axios";
+import { handleAPIErrors, getCloudCredentials, getLogger, isCloudAPIErrorResponse, retrieveApplicationName, DBOSCloudCredentials } from "../cloudutils.js";
+
+export async function updateApp(host: string, appName?: string, executorsMemoryMib?: number, userCredentials?: DBOSCloudCredentials): Promise<number> {
```

Comment:
Let's make this a separate command, not part of deploy

Reply:
done